src/pages/writing-nft/index.vue

```diff
@@ -256,6 +311,23 @@ export default {
     this.trendingClassIds = await this.filterNFTClassesByOwner(trendingClasses);
   },
   methods: {
+    getNFTModel,
+    handleNFTModelClick(e, classId) {
+      if (e.target.canActivateAR) {
+        e.target.activateAR();
+        return;
+      }
+      this.$nextTick(() => {
+        try {
```

Comment:
should use refs?
```js
this.$refs[classId]
```

Reply:
For which part you mean?

Reply:
replacing querySelector by Id

Reply:
Was planing to use use native id + hash to avoid js, turns out we still need js
Let's omit this for now, since this whole PR might get reverted next week anyway